eventhubs/client/azure-eventhubs/pom.xml

```diff
@@ -7,14 +7,14 @@
   <modelVersion>4.0.0</modelVersion>
   <parent>
     <groupId>com.azure</groupId>
-    <artifactId>azure-messaging-eventhubs-parent</artifactId>
-    <version>5.0.0-preview.1</version>
-    <relativePath>../pom.xml</relativePath>
+    <artifactId>azure-client-sdk-parent</artifactId>
+    <version>1.1.0</version>
+    <relativePath>../../../pom.client.xml</relativePath>
   </parent>
 
   <groupId>com.azure</groupId>
   <artifactId>azure-messaging-eventhubs</artifactId>
-  <version>5.0.0-preview.1</version>
+  <version>5.1.0-preview.1</version>
```

Comment:
Why is this going to 5.1.0 instead of 5.0.0-preview.2?

Reply:
Fixed